newIDE/app/src/Profile/AdditionalUserInfoDialog.js

```diff
@@ -108,20 +109,36 @@ const hearFromOptions = [
   },
 ];
 
-const selectFields = {
-  usage: {
-    title: <Trans>What are you using GDevelop for?</Trans>,
-    options: gdevelopUsageOptions,
-  },
-  creationExperience: {
-    title: <Trans>Have you used other game engines?</Trans>,
-    options: creationExperienceOptions,
-  },
-  hearFrom: {
-    title: <Trans>How did you hear about GDevelop?</Trans>,
-    options: hearFromOptions,
+const teacherOrTrainingCreationGoalOptions = [
+  {
+    value: 'teach-students',
+    label: `Teach students the principles of Game Development and logic`,
   },
-};
+  { value: 'train-workers', label: `Train workers on specific scenarios` },
+  { value: 'therapy', label: `Develop tools for therapy or rehab` },
```

Comment:
That seems quite specific. 🤔 
Should it be more about training the brain?

Reply:
Tagging @LuniMoon to see if we should change this, in the meantime I'll go ahead with this :)